src/system_info/system_info_sim_ivi.cc

```diff
@@ -23,11 +14,18 @@ SysInfoSim::SysInfoSim()
       msin_(""),
       spn_(""),
       conn_(NULL) {
-  InitDbusConnection();
+  conn_ = system_info::GetDbusConnection();
+  if (!conn_)
+    return;
+  modem_path_ = system_info::OfonoGetModemPath(conn_);
```

Comment:
forget my comment then. 
@liyingzh , please fix the `system_info::OfonoGetModemPath()` issue that @zolkis mentioned.